dateparser/custom_language_detection/map_languages.py

```diff
@@ -0,0 +1,18 @@
+from dateparser.data.languages_info import language_map
+
+
+def map_languages(language_codes):
+    """Returnes language supported candidates.
+
+    :param languages:
+        A list of language codes, e.g. ['en', 'es'] in ISO 639 Standard
+    :type languages: list
+
+    :return: Returns list[str] representing supported languages, else returns None
+    :rtype: list[str]
+    """
+    return_language_codes = []
+    for language_code in language_codes:
+        if language_code in language_map:
+            return_language_codes += language_map[language_code]
```

Comment:
@gavishpoddar @noviluni @lopuhin Do you think `extend()` is more pythonic here or `+=` is fine.

Reply:
Thanks, @kishan3, I will do that on the next commit